### ISSUE TITLE
The index page of simulators doesn't load

### ISSUE THREAD

**Issue by hghandri** [eureka2/G6K#19]
Hi,

I had some problem after installation with sources from "master" branch.
I've this message for example : "Impossible to access an attribute ("css") on a string variable ("Boîte à liste conforme RGAA")." when I tried to load index page

After investigation I've succeed to fix all error by modifying the source code.

Do you have similar problem ?

Best regards,

**Comment by eureka2** [comment]
Hi,

No, I did not have this problem!

Can you share your changes?

Best regards

**Comment by Honayme** [comment]
Hi, 

I'm also interested about your changes, let me know what they are.

Best regards

**Comment by eureka2** [comment]
@Honayme 
Can you describe what happens after the installation?

Best regards

**Comment by Honayme** [comment]
This is what happen when i run the project =>

Impossible to access an attribute ("css") on a string variable ("Boîte à liste conforme RGAA").
Ce simulateur n'existe pas ou n'est pas disponible
Veuillez réessayer ultérieurement

**Comment by eureka2** [comment]
Ok, tell me which version of g6k did you install, which version of Symfony the composer installer has installed, and which version of php you are using.

Is this the demo simulator you are testing?

**Comment by Honayme** [comment]
G6K : master branch
Symfony : 4.3.4 
Composer : 1.8.6 
Php : 7.3.2

I'm not aware of an eventual demo version . I cloned the master branch, follow the readme instructions and run the project for now. 

Maybe i miss something, I'm trying to figure out what.

Thx for your help

**Comment by eureka2** [comment]
Sorry, I can not reproduce this problem ;)
However, I installed PHP 7.3.9 and redo an installation from the master branch and everything works fine. I hope that @hghandri can help us!
```
PS C:\var\www\html> php -d memory_limit=-1 composer.phar create-project -s dev eureka2/g6k mysimu4-7/ 4.x-dev
Installing eureka2/g6k (4.x-dev 282eb11957431e6e8079c7cc1bbc76ee8cae1004)
  - Installing eureka2/g6k (4.x-dev 282eb11): Cloning 282eb11957 from cache
Created project in mysimu4-7/
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 113 installs, 0 updates, 0 removals
  - Installing symfony/flex (v1.4.6): Downloading (100%)
Symfony operations: 1 recipe (f8368c99b4c3e97ece7d7e36ec9c839f)
  - Configuring symfony/flex (>=1.0): From github.com/symfony/recipes:master
Loading composer repositories with package information
Updating dependencies (including require-dev)
Restricting packages listed in "symfony/symfony" to "^4.3"

Prefetching 6 packages
  - Downloading (100%)

Package operations: 112 installs, 0 updates, 0 removals
  - Installing ocramius/package-versions (1.4.0): Loading from cache
  - Installing eureka2/acroforms (1.0.6): Loading from cache
  - Installing flow/jsonpath (0.4.0): Loading from cache
  - Installing symfony/polyfill-mbstring (v1.12.0): Loading from cache
  - Installing twig/twig (v2.11.3): Loading from cache
  - Installing psr/log (1.1.0): Loading from cache
  - Installing symfony/polyfill-php73 (v1.12.0): Loading from cache
  - Installing symfony/debug (v4.3.4): Loading from cache
  - Installing symfony/polyfill-php72 (v1.12.0): Loading from cache
  - Installing symfony/polyfill-intl-idn (v1.12.0): Loading from cache
  - Installing symfony/mime (v4.3.4): Loading from cache
  - Installing symfony/http-foundation (v4.3.4): Loading from cache
  - Installing symfony/event-dispatcher-contracts (v1.1.5): Loading from cache
  - Installing psr/container (1.0.0): Loading from cache
  - Installing psr/cache (1.0.1): Loading from cache
  - Installing symfony/event-dispatcher (v4.3.4): Loading from cache
  - Installing symfony/http-kernel (v4.3.4): Loading from cache
  - Installing symfony/translation-contracts (v1.1.6): Loading from cache
  - Installing symfony/twig-bridge (v4.3.4): Loading from cache
  - Installing symfony/filesystem (v4.3.4): Loading from cache
  - Installing symfony/config (v4.3.4): Loading from cache
  - Installing symfony/twig-bundle (v4.3.4): Loading from cache
  - Installing symfony/translation (v4.3.4): Loading from cache
  - Installing symfony/intl (v4.3.4): Loading from cache
  - Installing symfony/polyfill-intl-icu (v1.12.0): Loading from cache
  - Installing symfony/service-contracts (v1.1.6): Loading from cache
  - Installing symfony/console (v4.3.4): Loading from cache
  - Installing symfony/finder (v4.3.4): Loading from cache
  - Installing symfony/routing (v4.3.4): Loading from cache
  - Installing symfony/dependency-injection (v4.3.4): Loading from cache
  - Installing symfony/var-exporter (v4.3.4): Loading from cache
  - Installing symfony/cache-contracts (v1.1.5): Loading from cache
  - Installing symfony/cache (v4.3.4): Loading from cache
  - Installing symfony/framework-bundle (v4.3.4): Loading from cache
  - Installing willdurand/js-translation-bundle (2.6.6): Loading from cache
  - Installing ashleydawson/simple-pagination (1.0.8): Loading from cache
  - Installing symfony/templating (v4.3.4): Loading from cache
  - Installing symfony/validator (v4.3.4): Loading from cache
  - Installing symfony/inflector (v4.3.4): Loading from cache
  - Installing symfony/property-access (v4.3.4): Loading from cache
  - Installing symfony/security-core (v4.3.4): Loading from cache
  - Installing symfony/security-http (v4.3.4): Loading from cache
  - Installing symfony/security-guard (v4.3.4): Loading from cache
  - Installing symfony/security-csrf (v4.3.4): Loading from cache
  - Installing symfony/security-bundle (v4.3.4): Loading from cache
  - Installing symfony/options-resolver (v4.3.4): Loading from cache
  - Installing symfony/form (v4.3.4): Loading from cache
  - Installing friendsofsymfony/user-bundle (v2.1.2): Loading from cache
  - Installing mobiledetect/mobiledetectlib (2.8.34): Loading from cache
  - Installing myclabs/deep-copy (1.9.3): Loading from cache
  - Installing setasign/fpdi (1.6.2): Loading from cache
  - Installing mpdf/mpdf (v7.1.9): Loading from cache
  - Installing doctrine/lexer (1.0.2): Loading from cache
  - Installing doctrine/annotations (v1.7.0): Loading from cache
  - Installing doctrine/reflection (v1.0.0): Loading from cache
  - Installing doctrine/event-manager (v1.0.0): Loading from cache
  - Installing doctrine/collections (v1.6.2): Loading from cache
  - Installing doctrine/cache (v1.8.0): Loading from cache
  - Installing doctrine/persistence (1.1.1): Loading from cache
  - Installing sensio/framework-extra-bundle (v5.4.1): Loading from cache
  - Installing symfony/asset (v4.3.4): Loading from cache
  - Installing symfony/expression-language (v4.3.4): Loading from cache
  - Installing symfony/lock (v4.3.4): Loading from cache
  - Installing symfony/stopwatch (v4.3.4): Loading from cache
  - Installing zendframework/zend-eventmanager (3.2.1): Loading from cache
  - Installing zendframework/zend-code (3.3.2): Loading from cache
  - Installing ocramius/proxy-manager (2.1.1): Loading from cache
  - Installing doctrine/dbal (v2.9.2): Loading from cache
  - Installing doctrine/inflector (v1.3.0): Loading from cache
  - Installing doctrine/common (v2.11.0): Loading from cache
  - Installing doctrine/migrations (2.1.1): Loading from cache
  - Installing symfony/doctrine-bridge (v4.3.4): Loading from cache
  - Installing doctrine/doctrine-cache-bundle (1.3.5): Loading from cache
  - Installing jdorn/sql-formatter (v1.2.17): Loading from cache
  - Installing doctrine/doctrine-bundle (1.11.2): Loading from cache
  - Installing doctrine/doctrine-migrations-bundle (v2.0.0): Loading from cache
  - Installing symfony/yaml (v4.3.4): Loading from cache
  - Installing doctrine/instantiator (1.2.0): Loading from cache
  - Installing doctrine/orm (v2.6.3): Loading from cache
  - Installing symfony/orm-pack (v1.0.6): Loading from cache
  - Installing symfony/serializer (v4.3.4): Loading from cache
  - Installing symfony/property-info (v4.3.4): Loading from cache
  - Installing webmozart/assert (1.5.0): Loading from cache
  - Installing phpdocumentor/reflection-common (2.0.0): Loading from cache
  - Installing phpdocumentor/type-resolver (1.0.1): Loading from cache
  - Installing phpdocumentor/reflection-docblock (4.3.2): Loading from cache
  - Installing symfony/serializer-pack (v1.0.2): Loading from cache
  - Installing egulias/email-validator (2.1.11): Loading from cache
  - Installing swiftmailer/swiftmailer (v6.2.1): Loading from cache
  - Installing symfony/swiftmailer-bundle (v3.2.8): Loading from cache
  - Installing psr/link (1.0.0): Loading from cache
  - Installing fig/link-util (1.0.0): Loading from cache
  - Installing symfony/web-link (v4.3.4): Loading from cache
  - Installing symfony/var-dumper (v4.3.4): Loading from cache
  - Installing symfony/web-profiler-bundle (v4.3.4): Loading from cache
  - Installing symfony/profiler-pack (v1.0.4): Loading from cache
  - Installing monolog/monolog (1.25.1): Loading from cache
  - Installing easycorp/easy-log-handler (v1.0.7): Loading from cache
  - Installing symfony/monolog-bridge (v4.3.4): Loading from cache
  - Installing symfony/monolog-bundle (v3.4.0): Loading from cache
  - Installing symfony/debug-bundle (v4.3.4): Loading from cache
  - Installing symfony/debug-pack (v1.0.7): Loading from cache
  - Installing symfony/dotenv (v4.3.4): Loading from cache
  - Installing nikic/php-parser (v4.2.4): Loading from cache
  - Installing symfony/maker-bundle (v1.13.0): Loading from cache
  - Installing symfony/phpunit-bridge (v4.3.4): Loading from cache
  - Installing symfony/css-selector (v4.3.4): Loading from cache
  - Installing symfony/dom-crawler (v4.3.4): Loading from cache
  - Installing symfony/browser-kit (v4.3.4): Loading from cache
  - Installing symfony/test-pack (v1.0.6): Loading from cache
  - Installing symfony/process (v4.3.4): Loading from cache
  - Installing symfony/web-server-bundle (v4.3.4): Loading from cache
Writing lock file
Generating autoload files
Symfony operations: 22 recipes (f8368c99b4c3e97ece7d7e36ec9c839f)
  - Configuring symfony/framework-bundle (>=4.2): From github.com/symfony/recipes:master
  - Configuring symfony/twig-bundle (>=3.3): From github.com/symfony/recipes:master
  - Configuring symfony/translation (>=3.3): From github.com/symfony/recipes:master
  - Configuring symfony/console (>=3.3): From github.com/symfony/recipes:master
  - Configuring symfony/routing (>=4.2): From github.com/symfony/recipes:master
  - Configuring willdurand/js-translation-bundle (>=2.0): From github.com/symfony/recipes:master
  - Configuring symfony/validator (>=4.3): From github.com/symfony/recipes:master
  - Configuring symfony/security-bundle (>=3.3): From github.com/symfony/recipes:master
  - Configuring friendsofsymfony/user-bundle (>=v2.1.2): From auto-generated recipe
  - Configuring doctrine/annotations (>=1.0): From github.com/symfony/recipes:master
  - Configuring sensio/framework-extra-bundle (>=5.2): From github.com/symfony/recipes:master
  - Configuring doctrine/doctrine-cache-bundle (>=1.3.5): From auto-generated recipe
  - Configuring doctrine/doctrine-bundle (>=1.6): From github.com/symfony/recipes:master
  - Configuring doctrine/doctrine-migrations-bundle (>=1.2): From github.com/symfony/recipes:master
  - Configuring symfony/swiftmailer-bundle (>=2.5): From github.com/symfony/recipes:master
  - Configuring symfony/web-profiler-bundle (>=3.3): From github.com/symfony/recipes:master
  - Configuring easycorp/easy-log-handler (>=1.0): From github.com/symfony/recipes:master
  - Configuring symfony/monolog-bundle (>=3.3): From github.com/symfony/recipes:master
  - Configuring symfony/debug-bundle (>=4.1): From github.com/symfony/recipes:master
  - Configuring symfony/maker-bundle (>=1.0): From github.com/symfony/recipes:master
  - Configuring symfony/phpunit-bridge (>=4.3): From github.com/symfony/recipes:master
  - Configuring symfony/web-server-bundle (>=3.3): From github.com/symfony/recipes:master
ocramius/package-versions:  Generating version class...
ocramius/package-versions: ...done generating version class
> App\G6K\Composer\ScriptHandler::buildDotenv
Creating the ".env" file
Enter the application environment [dev or prod] (dev):
Enter the debug mode [0 or 1] (0): 1
Enter the locale [en-GB, en-US, fr-FR, ...] (en-US): fr-FR
Enter the upload directory (%kernel.project_dir%/var/uploads):
Enter the absolute path of the pdftk executable (~): C:\Program Files (x86)\PDFtk\bin\pdftk.exe
Enter the mailer URL (null://localhost):
Enter the database engine [sqlite, mysql or pgsql] (sqlite):
Enter the database version (3.15):
Enter the database path (%kernel.project_dir%/var/data/databases/g6k.db):
Enter the HTTP proxy url [http://user:pass@host:port] (~):
Enter the HTTPS proxy url [https://user:pass@host:port] (~):
Executing script cache:clear [OK]
Executing script assets:install calcul [OK]
Executing script bazinga:js-translation:dump calcul/assets/bundles/bazingajstranslation/js [OK]
Executing script g6k:assets:manifest:refresh [OK]

> App\G6K\Composer\ScriptHandler::installUsers
Installing the users of the administration interface
> App\G6K\Composer\ScriptHandler::installDemo
Would you like to install the demo simulator? [y/N]
Installing the demo simulator

Executing script cache:clear [OK]
Executing script assets:install calcul [OK]
Executing script bazinga:js-translation:dump calcul/assets/bundles/bazingajstranslation/js [OK]
Executing script g6k:assets:manifest:refresh [OK]

Some files may have been created or updated to configure your new packages.
Please review, edit and commit them: these files are yours.

PS C:\var\www\html>
```
![image](https://user-images.githubusercontent.com/200478/65357209-b4025400-dbf6-11e9-87df-afc22b389671.png)

![image](https://user-images.githubusercontent.com/200478/65357391-3be85e00-dbf7-11e9-8792-5661026ac610.png)

**Comment by eureka2** [comment]
Ok I understood !
You have this message when you use the url `http://localhost/simulator/calcul/index.php` or `http://localhost/simulator/calcul/`
You do not normally have to use this url.
The urls to use are `http://localhost/simulator/calcul/admin` to access the administration interface or `http://localhost/simulator/calcul/<name of a simulator>` to access a simulator.
I will nevertheless make a correction to this problem.

**Comment by eureka2** [comment]
Fixed in version 4.3.5
